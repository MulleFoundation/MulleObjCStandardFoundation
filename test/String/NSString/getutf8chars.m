#import <MulleObjCStandardFoundation/MulleObjCStandardFoundation.h>

#include <stdio.h>
#include <stdlib.h>


static int   test( unsigned int size)
{
   NSString       *obj;
   NSUInteger     len;
   mulle_utf8_t   buf[ 256];
   char           *params_1[] =
   {
      "",
      "1848",
      "VfL Bochum",
      "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod "
      "tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim "
      "veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex "
      "ea commodi consequat. Quis aute iure reprehenderit in voluptate velit "
      "esse cillum dolore eu fugiat nulla pariatur." /* > 256 chars */,
      "L\xc3\xb6\xc3\xb6\xc3\xb6\xc3\xb6orem ipsum dolor sit amet, consectetur "
      "adipisici elit, sed eiusmod "
      "tempor incidunt ut lab\xc3\xb6re et d\xc3\xb6l\xc3\xb6re magna aliqua. "
      "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi "
      "ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in "
      "voluptate velit esse cillum dolore eu fugiat nulla pariatur." /* > 256 chars 16 bit*/,
      "L\xe2\x98\x84\xef\xb8\x8f\xe2\x98\x83\xef\xb8\x8f\xf0\x9f\x91\x8d\xf0\x9f\x8f\xbeorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod "
      "tempor incidunt ut lab\xc3\xb6re et d\xc3\xb6l\xc3\xb6re magna aliqua. "
      "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi "
      "ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in "
      "voluptate velit esse cillum dolore eu fugiat nulla pariatur." /* > 256 chars 32 bit*/
   };
   NSUInteger  i_1;
   NSUInteger  n_1 = sizeof( params_1) / sizeof( params_1[ 0]);

   assert( size < sizeof( buf));

   for( i_1 = 0; i_1 < n_1; i_1++)
   {
      @try
      {
         obj = [[[NSString alloc] initWithUTF8String:params_1[ i_1]] autorelease];
         len = [obj mulleGetUTF8Characters:buf
                                 maxLength:size];
         mulle_printf( "<%@> \"%.*@\": %td %td\n", [obj class], len, obj, [obj length], len);
      }
      @catch( NSException *localException)
      {
         printf( "*** Threw a %s exception\n", [[localException name] UTF8String]);
         return( 1);
      }
   }

   for( i_1 = 0; i_1 < n_1; i_1++)
   {
      @try
      {
         obj = [[[NSString alloc] mulleInitWithUTF8CharactersNoCopy:params_1[ i_1]
                                                             length:-1
                                                          allocator:NULL] autorelease];
         len = [obj mulleGetUTF8Characters:buf
                                 maxLength:sizeof( buf)];
         mulle_printf( "<%@> \"%.*@\": %td %td\n", [obj class], len, obj, [obj length], len);
      }
      @catch( NSException *localException)
      {
         printf( "*** Threw a %s exception\n", [[localException name] UTF8String]);
         return( 1);
      }
   }
   return( 0);
}


int   main( void)
{
   unsigned int  i;

   for( i = 2; i <= 128; i *= 2)
   {
      if( test( i))
         return( 1);
      if( test( i + 1))
         return( 1);
   }
   return( 0);
}

