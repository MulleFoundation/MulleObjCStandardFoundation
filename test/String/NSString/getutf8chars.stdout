<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "18": 4 2
<_MulleObjCTinyASCIIString> "Vf": 10 2
<_MulleObjCGenericASCIIString> "Lo": 318 2
<_MulleObjCAllocatorUTF16String> "L": 322 1
<_MulleObjCAllocatorUTF32String> "L": 324 1
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "184": 4 3
<_MulleObjCTinyASCIIString> "VfL": 10 3
<_MulleObjCGenericASCIIString> "Lor": 318 3
<_MulleObjCAllocatorUTF16String> "Lö": 322 3
<_MulleObjCAllocatorUTF32String> "L": 324 1
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL ": 10 4
<_MulleObjCGenericASCIIString> "Lore": 318 4
<_MulleObjCAllocatorUTF16String> "Lö": 322 3
<_MulleObjCAllocatorUTF32String> "L☄": 324 4
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL B": 10 5
<_MulleObjCGenericASCIIString> "Lorem": 318 5
<_MulleObjCAllocatorUTF16String> "Löö": 322 5
<_MulleObjCAllocatorUTF32String> "L☄": 324 4
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Boch": 10 8
<_MulleObjCGenericASCIIString> "Lorem ip": 318 8
<_MulleObjCAllocatorUTF16String> "Lööö": 322 7
<_MulleObjCAllocatorUTF32String> "L☄️": 324 7
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochu": 10 9
<_MulleObjCGenericASCIIString> "Lorem ips": 318 9
<_MulleObjCAllocatorUTF16String> "Löööö": 322 9
<_MulleObjCAllocatorUTF32String> "L☄️": 324 7
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolo": 318 16
<_MulleObjCAllocatorUTF16String> "Lööööorem ip": 322 16
<_MulleObjCAllocatorUTF32String> "L☄️☃️": 324 13
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor": 318 17
<_MulleObjCAllocatorUTF16String> "Lööööorem ips": 322 17
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍": 324 17
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, cons": 318 32
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit am": 322 32
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum ": 324 32
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, conse": 318 33
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit ame": 322 33
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum d": 324 33
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eius": 318 64
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, ": 322 64
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adip": 324 64
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusm": 318 65
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, s": 322 65
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipi": 324 65
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad": 318 128
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua.": 322 128
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre m": 324 128
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCTaggedPointerChar7String> "1848": 4 4
<_MulleObjCTinyASCIIString> "VfL Bochum": 10 10
<_MulleObjCGenericASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad ": 318 129
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. ": 322 129
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre ma": 324 129
<_MulleObjCTaggedPointerChar7String> "": 0 0
<_MulleObjCAllocatorASCIIString> "1848": 4 4
<_MulleObjCAllocatorASCIIString> "VfL Bochum": 10 10
<_MulleObjCAllocatorASCIIString> "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit i": 318 256
<_MulleObjCAllocatorUTF16String> "Lööööorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure repr": 322 256
<_MulleObjCAllocatorUTF32String> "L☄️☃️👍🏾orem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut laböre et dölöre magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis au": 324 256
